hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/SourceFormatAdapter.java

```diff
@@ -48,10 +56,120 @@
  */
 public final class SourceFormatAdapter implements Closeable {
 
+  public static class SourceFormatAdapterConfig extends HoodieConfig {
+    // sanitizes invalid columns both in the data read from source and also in the schema.
+    // invalid definition here goes by avro naming convention (https://avro.apache.org/docs/current/spec.html#names).
+    public static final ConfigProperty<Boolean> SANITIZE_AVRO_FIELD_NAMES = ConfigProperty
```

Comment:
minor. SANITIZE_SCHEMA_FIELD_NAMES

Reply:
you can change the config key accordingly as well

Reply:
yeah updated